packages/core/src/server.ts

```diff
@@ -1489,18 +1573,22 @@ export class Controller extends API {
             each.vaults = each.vaults.filter((v) => v.id !== vault.id);
         }
 
-        await this.updateMetaData(org);
+        await this.updateMetaData(org, account.id);
 
         // Save org
         await this.storage.save(org);
 
+        this.auditor.log("org", org.id, account.id, "updated", { accountId: account.id });
+
         // Delete vault
         await this.storage.delete(vault);
 
         this.log("vault.delete", {
             vault: { id: vault.id, name: vault.name, owner: vault.owner },
-            org: (org && { id: org.id, name: org.name, owner: org.owner }) || undefined,
```

Comment:
This was unnecessary (probably old copy+paste) because we're already tapping into the `org` before.